site/content/blogs/release-v133.en.md

```diff
@@ -33,11 +33,9 @@ Copilot v1.33 brings big enhancements to help you develop more flexibly and effi
 
 ## Use ECS Task Role for `copilot run local`
 
-`copilot run local` now has a flag `--use-task-role` which is enabled by default. When enabled, Copilot will retrieve the IAM permissions from your deployed service and inject them into the containers created by `run local`.
+`copilot run local` now has a flag `--use-task-role`. When enabled, Copilot will retrieve the IAM permissions from your deployed service and inject them into the containers created by `run local`.
```

Comment:
Do we need to explicitly mention that the flag is disabled by default?

Reply:
No, disabled by default is definitely the norm for flags in any program.